AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -509,8 +513,9 @@ - (void)upsertFromCosmosDbWithPartition:(NSString *)partition
   [self performCosmosDbOperationWithPartition:partition
                                    documentId:documentId
                                    httpMethod:kMSHttpMethodPost
-                                         body:body
+                                     document:document
```

Comment:
Is [this](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1514/files#diff-775692dad5e01b9791f9c25c9341fb31R507) still required?